include/openthread/thread_ftd.h

```diff
@@ -851,6 +851,24 @@ void otThreadSetCcmEnabled(otInstance *aInstance, bool aEnabled);
  */
 void otThreadSetThreadVersionCheckEnabled(otInstance *aInstance, bool aEnabled);
 
+/**
+ * Sets whether custom messages on the TMF port are allowed from the host.
```

Comment:
```suggestion
 * Enable or disable trusted host check for UDP messages on the TMF port, coming from the host.
```

This is just a possible suggestion to improve. The original line seems to imply that 'true' would be 'allow' and 'false' would be deny TMF-like messages, while it's actually the reverse. So it should refer to the enabling/disabling or setting of the "check" feature. Also "custom" is not well defined - it's about any UDP message with TMF destination port, if I'm correct. This may come from host application, host CLI, or an external IPv6 node, etc.